foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/asynchronous.processor.impl.py.vm

```diff
@@ -42,6 +42,16 @@ class ${step.capitalizedName}(${step.capitalizedName}Base):
         #end
         #end
 
+    #if ($step.isPersistTypeRdbms())
+    def get_data_source_configs(self) -> dict:
+        """
+        Retrieves the base configurations for the JDBC Connection
+        :return: JDBC Configs as a dictionary
+        """
+        configs = super().get_data_source_configs()
+        # TODO: Apply additional config options as needed by your specific database!
+        return configs
+    #end
```

Comment:
I: Overwritten class to give the user the option to extend database config properties if needed. This is primarily for databases that require more than the default jdbc connection requirements.